modules/management/kubectl-apply/main.tf

```diff
@@ -23,6 +23,11 @@ locals {
   apply_manifests_map = tomap({
     for index, manifest in var.apply_manifests : index => manifest
   })
+
+  install_kueue         = try(var.kueue.install, false)
+  install_jobset        = anytrue([try(var.jobset.install, false), local.install_kueue])
```

Comment:
- AFAIK Jobset is supported in Kueue but it is not a requirement for that.
- However, if we are going to install it with Kueue, it's better not to ignore user's explicit `false` input. Providing false can mean uninstalling Jobset.
- And if we are going to make Kueue's installation depended on Jobset, it's better to control it with precondition.

Reply:
In order to unblock this PR, I have made kueue and jobset installation independent. We can do this improvement in the next iteration.